### PR TITLE
Fix/basic repo

### LINKED ISSUE #1: Add path planning
Adding a path planning would really help in term of control, it would help going from one point to another without having to take care of the orientation the sub is before moving.

Read this doc : https://drive.google.com/open?id=1qZWxAjvj0-E1D_9l7iGvrZDU-IPU-6aV

### PR DESCRIPTION
**Pull request by supertoto29**
**Warning :** Before creating a pull request, make sure that it does not already exists in the [pull requests](../). Thank you.

## Description
Add the compiled files from control library. This add the ability to create trajectory by sending multiple pose.

## Issues
Closes : #1 

## How has this been tested ?
Tested with simulink with the interpreter and with the compiled code on vscode

## Checklist
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -2,5 +2,9 @@
     "python.autoComplete.extraPaths": [
         "/home/sonia/base_lib_ws/devel/lib/python2.7/dist-packages",
         "/opt/ros/melodic/lib/python2.7/dist-packages"
+    ],
+    "python.analysis.extraPaths": [
+        "/home/sonia/base_lib_ws/devel/lib/python2.7/dist-packages",
+        "/opt/ros/melodic/lib/python2.7/dist-packages"
```
Comment (Kam0ss) [inline .vscode/settings.json]: This should be noetic.

File: .vscode/settings.json [hunk 2]
```diff
@@ -2,5 +2,9 @@
     "python.autoComplete.extraPaths": [
         "/home/sonia/base_lib_ws/devel/lib/python2.7/dist-packages",
         "/opt/ros/melodic/lib/python2.7/dist-packages"
```
Comment (Kam0ss) [inline .vscode/settings.json]: This should be noetic.

**Review by Kam0ss — CHANGES_REQUESTED** [review]
A few changes to adapt for noetic, I am making the changes in the template.

File: .vscode/settings.json (hunk 2 above)
Comment (supertoto29) [reply .vscode/settings.json]: not sure we can edit this. These files are auto generated

Comment (Kam0ss) [reply .vscode/settings.json]: It's probably because you are still with python 2.7 on your OS. It might not be a problem then.

**Review by Kam0ss — APPROVED** [review]
LGTM

**Pull request merged**